FiducialRegistrationWizard/Logic/vtkPointDistanceMatrix.cxx

```diff
@@ -0,0 +1,200 @@
+#include "vtkPointDistanceMatrix.h"
+
+#include <vtkDoubleArray.h>
+#include <vtkMath.h>
+#include <vtkObjectFactory.h> //for new() macro
+
+//----------------------------------------------------------------------------
+vtkStandardNewMacro( vtkPointDistanceMatrix );
+
+//------------------------------------------------------------------------------
+vtkPointDistanceMatrix::vtkPointDistanceMatrix()
+{
+  pointList1 = vtkSmartPointer< vtkPoints >::New();
+  pointList2 = vtkSmartPointer< vtkPoints >::New();
+  distanceMatrix = vtkSmartPointer< vtkDoubleArray >::New();
+  UpdateNeededFlag = 0;
```

Comment:
If it works as a simple variable and not a pointer then it's even better, as it's simpler.